aspnetcore/release-notes/aspnetcore-9/includes/opt_out_metrics.md

```diff
@@ -0,0 +1,21 @@
+<!-- 
+[!INCLUDE[](~/release-notes/aspnetcore-9/includes/opt_out_metrics.md)]
+-->
```

Comment:
@tdykstra @wadepickett  I test my include in wn9 to make sure it looks right, then copy/paste the include here so no changes are made to wn9. That way there's no hurry to S&M
Even next sprint when we merge into a dotNet9RC1 branch, we'll want to avoid merge conflicts.

Reply:
@Rick-Anderson  and cc @tdykstra : 
Sounds great.  To keep this clear so we are not running over each other at the last second, who adds the includes in the What's new topic?  Would that be you Rick or all of us?  I assume you but I do not want to assume anything.

Also, We leave in the include lines we already added in the What's New topic for Prev 7 PR's, correct?

Reply:
Just a side note on adding and removing includes from the what's new topic for preview builds: The build here is currently out of sync with the actual changes.  It might be best to indicate when that is happening.  I was just looking through build and realized the change I was going to suggest was already in but had been added after the build.  I assume after the include was removed maybe.  I didn't look to see why it was not in sync.

Reply:
> Just a side note on adding and removing includes from the what's new topic for preview builds: The build here is currently out of sync with the actual changes. It might be best to indicate when that is happening. I was just looking through build and realized the change I was going to suggest was already in but had been added after the the last build. I assume after the include was removed maybe. I didn't look to see why it was not in sync.

There's no adding and removing to WN. You add include in your branch to test and then remove it when you S&M so WN doesn't change.

> @Rick-Anderson and cc @tdykstra : Sounds great. To keep this clear so we are not running over each other at the last second, who adds the includes in the What's new topic? Would that be you Rick or all of us? I assume you but I do not want to assume anything.
> 
> Also, We leave in the include lines we already added in the What's New topic for Prev 7 PR's, correct?

> ***who adds the includes in the What's new topic?***
Each of us adds our own includes to the WN topic. That only takes a few minutes and we'll synchronize so we don't get a merge conflict.